skiko/src/commonMain/kotlin/org/jetbrains/skiko/Events.kt

```diff
@@ -267,19 +267,20 @@ data class SkikoPointerEvent(
     val y: Double,
     val deltaX: Double = 0.0,
     val deltaY: Double = 0.0,
-    val buttons: SkikoMouseButtons = SkikoMouseButtons.NONE,
+    val pressedButtons: SkikoMouseButtons = SkikoMouseButtons.NONE,
+    val button: SkikoMouseButtons = SkikoMouseButtons.NONE,
     val modifiers: SkikoInputModifiers = SkikoInputModifiers.EMPTY,
     val kind: SkikoPointerEventKind,
     val timestamp: Long = 0,
     val platform: SkikoPlatformPointerEvent?
 )
 
 val SkikoPointerEvent.isLeftClick: Boolean
-    get() = buttons.has(SkikoMouseButtons.LEFT) && (kind == SkikoPointerEventKind.UP)
+    get() = pressedButtons.has(SkikoMouseButtons.LEFT) && (kind == SkikoPointerEventKind.UP)
```

Comment:
Probably we need to check `button` here instead of `pressedButtons`

Reply:
Done.